package.json

```diff
@@ -19,14 +19,14 @@
   "author": "Roi Avinoam",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/avinoamr/dbstream-memory/issues"
+    "url": "https://github.com/panoplyio/dbstream-memory/issues"
   },
-  "homepage": "https://github.com/avinoamr/dbstream-memory",
+  "homepage": "https://github.com/panoplyio/dbstream-memory",
   "devDependencies": {
     "mocha": "^1.21.0"
   },
   "dependencies": {
-    "dbstream": "git+https://github.com/avinoamr/dbstream",
+    "dbstream": "https://github.com/panoplyio/dbstream#v1.0.3",
```

Comment:
Do you know what the `git+` prefix does?

We also forgot to bump the version in the `dbstream/package.json`, although I believe it's not so critical:

<img width="474" alt="Screen Shot 2022-08-31 at 6 54 43 PM" src="https://user-images.githubusercontent.com/43310906/187723613-698cbb32-447d-473b-8fb2-ab098e6b12c1.png">

Reply:
See https://github.com/panoplyio/dbstream-mongo/pull/8#discussion_r960551092